crates/schema/src/def.rs

```diff
@@ -84,7 +85,9 @@ pub struct ModuleDef {
     tables: IdentifierMap<TableDef>,
 
     /// The reducers of the module definition.
-    reducers: IdentifierMap<ReducerDef>,
+    /// Note: this is using IndexMap because reducer order is important
```

Comment:
Can we get a corresponding comment on `RawModuleDefv9` as well

Reply:
Where do you mean? I didn't change its type.

Reply:
For most of the vectors in that struct, order does not matter, the contained data is sorted. However, this is no longer true for the `reducers` vector, whose order is now semantically important.

Reply:
Actually, this sort of doesn't matter, since the module itself never uses the Reducer ID.